src/app/models/test-result.model.ts

```diff
@@ -1,19 +1,21 @@
 import { TestType } from '../models/test.type';
-import { BodyType } from './body-type';
+import { BodyType } from '@app/models/body-type';
 
 export interface TestResultModel {
+  reasonForCreation: string;
+  testResultId: string;
   testerStaffId: string;
-  testStartTimestamp: Date;
+  testStartTimestamp: string;
   odometerReadingUnits: string;
-  testEndTimestamp: Date;
+  testEndTimestamp: string;
```

Comment:
Hi @JonLazariniDD, I suggested to use the string type for dates partly because 
- the API specs specifies them as string format
- it will be easy to parse and implement
- but the onus is on the dev to ensure the string format is returned to the back end as expected i.e `YYYY-MM-DD`, `YYYY-MM-DD hh:mm:ss`